.github/workflows/bind-schemas-to-projects.yml

```diff
@@ -0,0 +1,166 @@
+    
+    - name: Determine schema version
+      id: version
+      run: |
+        if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+          echo "version=${{ github.event.inputs.schema_version }}" >> $GITHUB_OUTPUT
+        else
+          # Extract version from repository dispatch event
+          VERSION=${{ github.event.client_payload.version }}
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+        fi
+        echo "Schema version: ${{ steps.version.outputs.version }}"
+    
+    - name: Download schemas from htan2-data-model release
+      run: |
+        # Create schemas directory
+        mkdir -p schemas
+        
+        # Download schemas from the htan2-data-model release
+        SCHEMA_VERSION=${{ steps.version.outputs.version }}
+        REPO=${{ github.event.inputs.data_model_repo || 'sage-bionetworks/htan2-data-model' }}
+        
+        echo "Downloading schemas from $REPO release $SCHEMA_VERSION"
+        
+        # Get the release assets
+        RELEASE_URL="https://api.github.com/repos/$REPO/releases/tags/$SCHEMA_VERSION"
+        echo "Release URL: $RELEASE_URL"
+        
+        # Download each schema file
+        for schema_file in Demographics Diagnosis Therapy FollowUp MolecularTest Exposure FamilyHistory VitalStatus BulkWESLevel1; do
+          echo "Downloading $schema_file schema..."
+          curl -L -H "Accept: application/vnd.github.v3.raw" \
+            "https://api.github.com/repos/$REPO/contents/JSON_Schemas/HTAN.$schema_file-$SCHEMA_VERSION-schema.json?ref=$SCHEMA_VERSION" \
+            -o "schemas/HTAN.$schema_file-$SCHEMA_VERSION-schema.json" || echo "Schema $schema_file not found, skipping..."
+        done
+        
+        # List downloaded schemas
+        echo "Downloaded schemas:"
+        ls -la schemas/
+    
+    - name: Bind file-based schemas to project subfolders
+      env:
+        SYNAPSE_USERNAME: ${{ secrets.SYNAPSE_USERNAME }}
+        SYNAPSE_AUTH_TOKEN: ${{ secrets.SYNAPSE_AUTH_TOKEN }}
+      run: |
+        echo "Binding schemas to project subfolders..."
+        
+        # Load project configuration
+        echo "Loading project configuration from projects.yml and schema_binding_config.yml..."
+        
+        # Find WES Level 1 schema (file-based)
+        WES_SCHEMA=$(find schemas -name "*WES*Level*1*" -o -name "*BulkWESLevel1*" | head -1)
+        
+        if [ -n "$WES_SCHEMA" ]; then
+          echo "Found WES Level 1 schema: $WES_SCHEMA"
+          
+          # Bind to HTAN2_Ovarian project WES_Level_1 subfolder
+          echo "Binding to HTAN2_Ovarian WES_Level_1 subfolder (syn69630475)..."
+          
+          poetry run python scripts/synapse_json_schema_bind.py \
+            -p "$WES_SCHEMA" \
+            -t "syn69630475" \
+            -n "${{ env.ORGANIZATION_NAME }}" \
+            --create_fileview
+        else
+          echo "No WES Level 1 schema found, skipping file-based binding"
+        fi
```

Comment:
Am I right in my reading that this both registeres the schema and then binds it.

What happens as a schema gets registered multiple times in sequences the script binds to different containers.

Should this be cut down to only bind the schema and not also register?